### ISSUE TITLE
jog_arm first order low-pass filter

### ISSUE THREAD

**Issue by AndyZe** [ros-planning/moveit#1471]
Per conversation in #1468, the 2nd-order filter that is currently used is known to overshoot. A first-order filter may be a better fit (as well as more efficient).

https://dankelley.github.io/r/2014/01/15/butterworth-filters.html

**Comment by SansoneG** [comment]
Changing
```cpp
double new_filtered_msrmt =
     (1. / (1. + filter_coeff_ * filter_coeff_ + 1.414 * filter_coeff_)) *
     (previous_measurements_[2] + 2. * previous_measurements_[1] + previous_measurements_[0] -
      (filter_coeff_ * filter_coeff_ - 1.414 * filter_coeff_ + 1.) * previous_filtered_measurements_[1] -
      (-2. * filter_coeff_ * filter_coeff_ + 2.) * previous_filtered_measurements_[0]);
```
to
```cpp
double new_filtered_msrmt =
    (1. / (1. + filter_coeff_)) *
    (previous_measurements_[1] + previous_measurements_[0] -
     (-1. * filter_coeff_ + 1.) * previous_filtered_measurements_[0]);
```
in `low_pass_filter.cpp:66-70` seems to solve the issue.

Here is a plot of the new filter:
![Bw_lp_1](https://user-images.githubusercontent.com/50736605/58411744-480f4500-8075-11e9-901b-e83020cf6044.png)

**Comment by jschleicher** [comment]
@SansoneG Please provide a pull-request.

**Comment by SansoneG** [comment]
> @SansoneG Please provide a pull-request.

#1483

**Comment by rhaschke** [comment]
Fixed in #1483.
